README.md

```diff
@@ -3,12 +3,16 @@
 [![web-prover workflow](https://github.com/pluto/web-prover/actions/workflows/web-prover.yaml/badge.svg)](https://github.com/pluto/web-prover/actions/workflows/web-prover.yaml)
 [![docs](https://img.shields.io/badge/docs-e28f00)](https://docs.pluto.xyz)
 
-Pluto's Web Prover repo contains the source code for three types of [Web Proofs](https://pluto.xyz/blog/introducing-pluto#1927a922ef2980748958f9f1fa514320):
+The Web Prover is infrastructure for generating Web Proofs. There are 2 components of the Web Prover — (1) the Client Crates, which run inside of the app or browser; and (2) Notary Crate, which is a hosted service. The default option for generating Web Proofs is to use the Pluto-hosted service.
```

Comment:
It's 3 components: Clients, Notary, and Proofs

Reply:
made edit to fix this